packages/demo/src/configuration/blocks/effects/compositionBlock.ts

```diff
@@ -146,11 +146,11 @@ export class CompositionShaderBinding extends ShaderBinding {
         effect.setTexture(this.getRemappedName("background"), background);
         effect.setTexture(this.getRemappedName("foreground"), foreground);
 
-        if (foreground) {
```

Comment:
A similar change to this is needed for BlurBlock, which I think checks if the input texture exists before setting some width/height variables

Reply:
We've already disabled optimization of the blur block since it needs to be multi-pass